_content/av/forgotten-temple-of-banteay-chhmar.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "The Forgotten Temple of Banteay Chhmar"
+publisher: "Deutsche Welle"
```

Comment:
In the credits at the end you'll see the authorship info. Add "Wolfgang Luck" as the author and "Karl-Heinz Satzger" as editor.